liblumen_alloc/src/erts/process/test.rs

```diff
@@ -241,6 +242,41 @@ fn simple_gc_test(process: Process) {
     assert!(list_term.is_list());
     let list_ptr = list_term.list_val();
 
+    // Allocate a closure with an environment [101, "this is a binary"]
+    let closure_num = Term::make_smallint(101);
+    let closure_string = "this is a binary";
+    let closure_string_term = process.binary_from_str(closure_string).unwrap();
+
+    let creator = Term::NIL;
```

Comment:
This feels wrong as the creator should be a Pid term.

Reply:
My thoughts was that it wouldn't matter in this context, but you're right, makes much more sense to make it a Pid.